docs/templates/eia861_child.rst.jinja

```diff
@@ -0,0 +1,39 @@
+{% extends "data_source_parent.rst.jinja" %}
+
+{% block background %}
+EIA Form EIA-861 (and the short form EIA-861S) make up the Annual Electric Power
+Industry Report, collecting data from distribution utilities and power marketers.
+It is a census of all United States electric utilities. As of 2023 PUDL
+only incorporates the annual data, but there's also a less detailed monthly `Form
+EIA-861M <https: //www.eia.gov/electricity/data/eia861m />`__.
+
+There are more than 20 different spreadsheets included in the annual zipfiles. For
+details, see the `official EIA-861 page <{{ source.path }}>`__
+{% endblock %}
+
+{% block downloadable_pdfs %}
+{% for filename in download_paths %}
+* :download:`{{ filename.stem.replace("_", " ").title() }} (PDF) <{{ filename }}>`
+{% endfor %}
+{% endblock %}
+
+{% block accessible %}
+{% endblock %}
+
+{% block fill_out_form %}
+{% endblock %}
+
+{% block original_data %}
+{% endblock %}
+
```

Comment:
This page is basic right now, but at least it exists. We can fill it out more as we work on the EIA-861.